Documentation/community/governance/commit_access.rst

```diff
@@ -98,15 +98,14 @@ can ask the original submitter to address it. Regardless, you need to
 ensure that the problem is fixed in a timely way. The definition of
 "timely" depends on the severity of the problem.
 
-If a bug is present on master and other branches, fix it on master
-first, then backport the fix to other branches. Straightforward
-backports do not require additional review (beyond that for the fix on
-master).
+If a bug is present on main and other branches, fix it on main first,
+then backport the fix to other branches. Straightforward backports
+do not require additional review (beyond that for the fix on main).
```

Comment:
Just want to call this out as a great example of straightforward, clear writing. ✨

Reply:
I'll pass those kudos back to the community :-) In this PR, I'm just a fancy version of `sed`.